ext/crates/fp/src/vector/internal/mod.rs

```diff
@@ -671,13 +671,15 @@ pub trait InternalBaseVectorMutP<const P: u32>: InternalBaseVectorP<P> {
         self._limbs_mut()[target_range.end - 1] |= result;
     }
 
-    // The following methods are only implemented on `FpVectorP` right now. This is the only use
-    // case so far, so I don't feel too bad about marking them as unimplemented.
-
+    /// This method is only implemented on `FpVectorP` right now. This is the only use case so far,
+    /// so I don't feel too bad about marking them as unimplemented.
     fn _copy_from_slice(&mut self, _slice: &[u32]) {
         unimplemented!();
     }
 
+    /// This function is currently only used in [`algebra::PolynomialAlgebra::multiply_monomials`],
+    /// on [`FpVectorP`]. We mostly care about the Steenrod algebra, which doesn't use this code, so
+    /// I'm marking it as unimplemented for now.
```

Comment:
@hoodmane should confirm but my understanding is that PolynomialAlgebra is useful for implementing Steenrod modules that are polynomial algebras (e.g. cohomology of BU(n)).

Reply:
Yeah. This is in particular used for divided power algebras and exterior algebras. The name `PolynomialAlgebra` really means a tensor of divided power, exterior, and polynomial parts. It was intended for these to also have streenrod algebra structure. Once upon a time I had the Dixon algebra because of a special request, but doing BU(n) and U(n) is more reasonable. It's worth noting though that I never quite got the sign rule correct at odd primes.

Also it looks like the definition of `add_truncate` isn't even currently correct, probably because the name is cryptic and it has no test coverage.

Reply:
Does that mean that I should implement `add_truncate` anyway? I'm not concerned with other modules than F_2 personally but I wouldn't want to break the code for someone who uses ext to compute BU(n)

Reply:
Since it's already broken it doesn't seem to me that this PR is making the situation any worse.

Reply:
Ah, got it. I thought it was only broken for odd primes. I could even remove the dead code from the limbs module then

Reply:
Actually I take it back it works correctly I was just reading the code wrong.

Reply:
Alright, I added the implementation when the source is an `FpVector` and the target starts on a limb boundary